Gems/ScriptCanvas/Code/Editor/Nodes/NodeUtils.h

```diff
@@ -51,9 +51,9 @@ namespace ScriptCanvasEditor
             GeneralNode
         };
 
-        struct NodeConfiguration
+        struct NodeReplacementConfiguration
```

Comment:
why do you rename this as replacement configuration?

Reply:
as apposed to configuring the node itself